core/src/main/scala/org/apache/spark/resource/ResourceUtils.scala

```diff
@@ -362,8 +362,9 @@ private[spark] object ResourceUtils extends Logging {
 
   def logResourceInfo(componentName: String, resources: Map[String, ResourceInformation])
     : Unit = {
+    if (resources.isEmpty) return
     logInfo("==============================================================")
-    logInfo(s"Resources for $componentName:\n${resources.mkString("\n")}")
+    logInfo(s"Accelerate Resources for $componentName:\n${resources.mkString("\n")}")
```

Comment:
@tgravescs Do we always log all the resources after introducing `ResourceProfile`? If so, I think we may could remove `logResourceInfo`.

Reply:
I'm not sure what your question is, you can see where this is called from. - both driver and executors. This was before the REsourceProfile was introduced. I chose to leave it even when empty for consistency in reading the logs and I don't see it hurts anything and it just tells you nothing was set, but if you want it removed I'm ok with that as well.